.github/workflows/maven-test.yml

```diff
@@ -2,28 +2,17 @@ name: Test with Maven
 
 on:
   workflow_dispatch:
+  push:
+    branches:
+     - 'develop'
+     - 'master'
+     - 'releases/**'
   pull_request:
-    branches: [ "develop" ]
+    branches:
+      - 'develop'
+      - 'master'
+      - 'releases/**'
```

Comment:
Note that the mobile devs use `release` name for release branches instead of `releases`:
https://github.com/wultra/powerauth-mobile-sdk/branches

In new repositories, `master` branch will be replaced by `main` branch.

Not a problem at all because we can configure branch names easily, just pointing out the little differences.